app/src/main/res/values/dimens.xml

```diff
@@ -18,217 +18,317 @@
-  <dimen name="profile_chooser_add_view_parent_margin_end_profile_not_added">298dp</dimen>
   <dimen name="profile_chooser_profile_view_parent_margin_start_profile_not_added">298dp</dimen>
   <dimen name="profile_chooser_profile_view_parent_margin_end_profile_not_added">298dp</dimen>
+  <dimen name="profile_chooser_profile_view_parent_padding">4dp</dimen>
+
+  <!-- Profile Chooser Profile View Is Admin Margin Top -->
+  <dimen name="profile_chooser_profile_view_is_admin_margin_top_profile_already_added">8dp</dimen>
+  <dimen name="profile_chooser_profile_view_is_admin_margin_top">4dp</dimen>
+
+  <!-- Profile Chooser Profile View Last Visited -->
+  <dimen name="profile_chooser_profile_view_last_visited_margin_top_profile_not_added">4dp</dimen>
+  <dimen name="profile_chooser_profile_view_last_visited_margin_start_profile_not_added">12dp</dimen>
+  <dimen name="profile_chooser_profile_view_last_visited_margin_top_profile_already_added">8dp</dimen>
+
+  <!-- Profile Chooser Margin -->
+  <dimen name="profile_chooser_margin_start_profile_already_added">72dp</dimen>
+  <dimen name="profile_chooser_margin_end_profile_already_added">72dp</dimen>
+  <dimen name="profile_chooser_margin_top_profile_already_added">0dp</dimen>
+  <dimen name="profile_chooser_margin_top_profile_not_added">84dp</dimen>
+
+  <!-- Profile Chooser Administrator Controls -->
+  <dimen name="profile_chooser_administrator_controls_margin_bottom">28dp</dimen>
+
+  <!-- Profile Chooser Setting Icon -->
+  <dimen name="profile_chooser_setting_icon_margin_end">96dp</dimen>
+
+  <!-- Profile Chooser Profile -->
+  <dimen name="profile_chooser_profile_select_text_margin_start">96dp</dimen>
+  <dimen name="profile_chooser_padding_bottom_profile_already_added">224dp</dimen>
+
+<!--  Profile Chooser: Add View-->
+  <dimen name="profile_chooser_add_view_margin_bottom_profile_not_added">28dp</dimen>
+  <dimen name="profile_chooser_add_view_margin_start_profile_already_added">16dp</dimen>
+  <dimen name="profile_chooser_add_view_margin_end_profile_already_added">16dp</dimen>
+  <dimen name="profile_chooser_add_view_margin_top_profile_already_added">16dp</dimen>
+  <dimen name="profile_chooser_add_view_margin_top_profile_not_added">60dp</dimen>
+  <dimen name="profile_chooser_add_view_margin_end_profile_not_added">24dp</dimen>
+  <dimen name="profile_chooser_add_view_margin_start_profile_not_added">24dp</dimen>
+  <dimen name="profile_chooser_add_view_parent_margin_start_profile_not_added">298dp</dimen>
+  <dimen name="profile_chooser_add_view_parent_margin_end_profile_not_added">298dp</dimen>
+  <dimen name="profile_chooser_add_view_circular_image_margin_top_profile_already_added">48dp</dimen>
+
+  <!-- Profile Chooser Description Margin -->
+  <dimen name="profile_chooser_description_margin_start_profile_not_added">24dp</dimen>
+  <dimen name="profile_chooser_description_margin_end_profile_not_added">24dp</dimen>
```

Comment:
Nit: Here and elswewhere in the file, it is enough to categorize the dimens by screen name, and no need to further categorize them, so you can delete the additional comments in the newlines, and keep the values with the same screenname together.

Reply:
okay, done